src/drivers/batt_smbus/batt_smbus.cpp

```diff
@@ -57,9 +57,8 @@ BATT_SMBUS::BATT_SMBUS(I2CSPIBusOption bus_option, const int bus, SMBus *interfa
 	int battsource = 1;
 	int batt_device_type = (int)SMBUS_DEVICE_TYPE::UNDEFINED;
 
-	param_set(param_find("BAT_SOURCE"), &battsource);
-	param_get(param_find("BAT_SMBUS_MODEL"), &batt_device_type);
-
+	param_set(param_find("BAT1_SOURCE1"), &battsource);
```

Comment:
need to be BAT1_SOURCE
is there chance not to use param_find()? and use 'px4::params::BAT1_SOURCE'?

Reply:
Moving to `px4::params::BAT1_SOURCE'` would be cleaner right now, but might be a (small) step in the wrong direction if there's any interest in supporting multiple instances of batt_smbus.

Reply:
and what about "BAT1_SOURCE1" -> "BAT1_SOURCE"

Reply:
Fixed.